template/src/features/counter.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const slice = createSlice({
+  name: 'counter',
+  initialState: {
+    value: 0,
+  },
+  reducers: {
+    increment: state => {
+      state.value += 1;
+    },
+    decrement: state => {
+      state.value -= 1;
+    },
+  },
+});
+
+const selectCount = state => state.counter.value;
+
+export const selectors = {
+  selectCount,
+};
+
+export const { actions } = slice;
```

Comment:
Export the action creators individually here:

```suggestion
export const { increment, decrement} = slice.actions;
```